notebooks/multi_modal/solutions/image_captioning.ipynb

```diff
@@ -0,0 +1,1136 @@
+{
```

Comment:
<p>Can add this to the description? "Start and end tokens are added here because we are using an encoder-decoder model and during prediction, to get the captioning started we use &lt;start&gt; and since captions are of variable length, we terminate the prediction when we see the &lt;end&gt; token." </p><p> </p><p>Let me know if that makes sense.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>That looks very clear! Added it. Thanks!</p>